matrix-sdk-android/src/test/java/org/matrix/android/sdk/api/session/pushrules/PushRulesConditionTest.kt

```diff
@@ -155,6 +172,33 @@ class PushRulesConditionTest : MatrixTest {
         }
     }
 
+    @Test
+    fun test_decrypted_eventmatch_type_condition() {
```

Comment:
add some test for content match rules:
- `m.rules.contains_user_name`: content.body matches "alice"
- `m.rules.roomnotif`: content.body matches "@room"

And an additional test to proove that a decrypted event won't match anymore the `.m.rule.encrypted`(type matches "m.room.encrypted")